plugins/coverbrowser.koplugin/main.lua

```diff
@@ -161,58 +162,132 @@ function CoverBrowser:addToMainMenu(menu_items)
     -- add Mosaic / Detailed list mode settings to File browser Settings submenu
     -- next to Classic mode settings
     if menu_items.filebrowser_settings == nil then return end
-    table.insert (menu_items.filebrowser_settings.sub_item_table, 4, {
+    table.insert (menu_items.filebrowser_settings.sub_item_table, 5, {
```

Comment:
Screenshot of the upper menu where we see this reordering please?

Reply:
The order was wrong after adding "Show finished books".